app/.env.development

```diff
@@ -0,0 +1,7 @@
+# !! Don't put secrets in this file. Use .env.local for secrets instead !!
+# This file is checked into your git repo and provides defaults for non-sensitive
+# env vars when running `next dev` only.
+# Learn more: https://nextjs.org/docs/app/building-your-application/configuring/environment-variables
+
+# If you deploy to a subpath, change this to the subpath so relative paths work correctly.
+NEXT_PUBLIC_NEXT_PUBLIC_BASE_PATH=
```

Comment:
how come this file is needed now but wasn't needed when it was just BASE_PATH?

Reply:
This was something extra I included, alongside the rename. In #145 I'm including Next.js files that reference the environment variable, and I was worried without providing an example of how it would be set, that it would be confusing about where it is coming from for the app.